src/core/base-project.ts

```diff
@@ -87,6 +87,7 @@ export default abstract class BaseProject {
     async deleteFolder(): Promise<void> {
         if (fs.existsSync(this.projectPath)) {
             AppLogger.debug(`Removing folder '${this.projectPath}'`, true);
+            console.log(this.projectPath,"<<<<<<<<this.projectPath")
```

Comment:
Hi @RohitBhosale204 could you please remove these console logs

Reply:
Yes